src/listeners.ts

```diff
@@ -26,9 +27,16 @@ async function processCommit(
 export async function onPush(
   context: Context<Webhooks.WebhookPayloadPush>
 ): Promise<void> {
+  addInvocationBreadcrumb("'onPush' Github Webhook push event handler");
   const { commits, repository } = context.payload;
   const commitInfos = await Promise.all(
-    commits.map(async (commit) => processCommit(commit, repository.full_name))
+    commits.map(async (commit: GithubCommit) => {
+      context.log.info("Received new #commit: %O", {
```

Comment:
This way the object is included within the message. That object should be passed as the first parameter, so the log entry JSON would look like this:
```json
{
    "msg": "Received new commit",
    "repoName": "...",
    "hash": "..."
}
```